cmd/stack_package.go

```diff
@@ -195,7 +197,18 @@ The packaging process builds the stack image, generates the "tar.gz" archive fil
 
 			// docker build
 			// create the image name to be used for the docker image
-			namespaceAndRepo := imageNamespace + "/" + stackID
+			imageRepo := rootConfig.CliConfig.GetString("images")
+
+			if imageNamespace == "dev.local" && imageRegistry != "docker.io" {
```

Comment:
Hi Sandy. I've been thinking about this more and talking to Michele. We've really been using `dev.local` wrong (not your fault). 
Keeping in mind that the image format is REGISTRY/NAMESPACE/REPOSITORY:TAG
The `dev.local` part should really be the registry, not the namespace. It is implying that this is a non-existing, local only registry. 

Please investigate making the `--image-registry` default to `dev.local` and the `--image-namespace` default to something else, like `stacks`. I haven't fully thought through all the coding challenges of this, so please try it out and let us know what this complicates and/or simplifies.

Reply:
I spoke with @neeraj-laad about this and it makes sense to have `dev.local` as the registry. `appsody` might be a better default option for the namespace over `stacks`